MapleServer2/Data/Static/MapMetadataStorage.cs

```diff
@@ -146,8 +147,21 @@ public static int GetDistanceToNextBlockBelow(int mapId, CoordS coord, out MapBl
         return 99;
     }
 
-    public static bool MapIsInstancedOnly(int mapId)
+    public static bool IsInstancedOnly(int mapId)
     {
+        if (mapId is >= 50000001 and <= 50000014) // lobbyIds are capacity > 0, yet are instanced only
```

Comment:
it has to be an better way to handle these lobbies

Reply:
perhaps, I don't know one